app/models/interactive_page.rb

```diff
@@ -264,32 +257,17 @@ def export
                                     :embeddable_display_mode,
                                     :additional_sections,
                                     :is_completion])
-
-    page_json[:interactives] = []
     page_json[:embeddables] = []
-    page_json[:sections] = []
 
-    self.interactives.each do |inter|
-      interactive_hash = helper.wrap_export(inter)
-      if inter.respond_to? :linked_interactive
-        interactive_hash[:linked_interactive] = inter.linked_interactive.present? ? helper.wrap_export(inter.linked_interactive) : nil
-      end
-      page_json[:interactives] << interactive_hash
-    end
-    self.main_embeddables.each do |embed|
+    self.embeddables.each do |embed|
       embeddable_hash = helper.wrap_export(embed)
-      page_json[:embeddables] << embeddable_hash
-    end
-    self.class.registered_additional_sections.each do |s|
-      additional_section = {name:s[:name],section_embeddables:[]}
-      self.section_embeddables(s[:name]).each do |embed|
-        section_embeddable_hash = helper.wrap_export(embed)
-        additional_section[:section_embeddables] << section_embeddable_hash
+      if embed.respond_to? :linked_interactive
+        embeddable_hash[:linked_interactive] = embed.linked_interactive.present? ? helper.wrap_export(embed.linked_interactive) : nil
       end
-      page_json[:sections] << additional_section
+      page_json[:embeddables] << { embeddable: embeddable_hash, section: embed.page_section }
```

Comment:
I think this format change will break what @dougmartin  is using to generate "prompts" for Collabspace. I think that is OK.  But we probably want to add a version number to the export, so Doug or other systems can check this, so they know how to handle the exported file.